city_scrapers/spiders/cook_justice_advisory.py

```diff
@@ -0,0 +1,196 @@
+        """
+        Parse or generate all-day status. Defaults to false.
+        """
+        date = response.xpath(
+            '//span[@class="date-display-single"]/descendant-or-self::*/text()'
+        ).extract()
+        date = "".join(date).upper()
+        return "ALL DAY" in date
+
+    def _parse_title(self, response):
+        """Parse or generate event"""
+        title = "".join(response.css("h1::text").extract())
+        if "JAC Council Meeting" in title:
+            return "JAC Council Meeting"
+        else:
+            return title
+
+    def _parse_description(self, response):
+        """Parse or generate event description."""
+        category_field = response.xpath(
+            "//div[contains(., 'Category:') and contains(@class, 'field-label')]"
+        )
+        field_items = category_field.xpath(
+            "./following::div[contains(@class, 'field-items')]"
+        )
+        return " ".join(
+            field_items.xpath(".//p/text()").extract()
+            + field_items.xpath(".//strong/text()").extract()
+        ).strip()
+
+    def _parse_start(self, response):
+        """Parse start date and time"""
+        start = response.xpath(
+            '//span[@class="date-display-single"]/descendant-or-self::*/text()'
+        ).extract()
+        start = "".join(start).upper()
+        start = start.split(" TO ")[0].strip()
+        start = start.replace("(ALL DAY)", "12:00AM")
+
+        return datetime.strptime(start, "%B %d, %Y %I:%M%p")
+
+    def _parse_end(self, response):
+        """Parse end date and time"""
+        date = response.xpath(
+            '//span[@class="date-display-single"]/descendant-or-self::*/text()'
+        ).extract()
+        date = "".join(date).upper()
+        date.replace("(ALL DAY)", "TO 11:59PM")
+        start_end = date.split(" TO ")
+
+        if len(start_end) < 2:
+            return
+
+        end_time = start_end[1]
+        date = start_end[0][: start_end[0].rindex(" ")]
+        return datetime.strptime("{} {}".format(date, end_time), "%B %d, %Y %I:%M%p")
+
+    def _parse_links(self, response):
+        """Parse links"""
+        links = response.css("span.file a")
+        links = links[2:]
+        for link in links:
+            link_name = link.xpath("text()").extract_first()
+            link_name = link_name.replace("\xa0", " ")
+            link_path = link.xpath("./@href").extract_first()
+            pattern = r"(?P<m>[a-zA-Z]+)( *)(?P<y>\d{4})"
+            regex = re.search(pattern, link_name)
+            if regex is not None:
+                raw_monthyear = regex.group("m") + " " + regex.group("y")
+                if len(regex.group("m")) < 4:
+                    date_obj = datetime.strptime(raw_monthyear, "%b %Y")
+                else:
+                    date_obj = datetime.strptime(raw_monthyear, "%B %Y")
+                formatted_date = datetime.strftime(date_obj, "%y-%m")
+                self.agenda_map[formatted_date] = [
```

Comment:
Mentioned above, but looking again I'm realizing there are meeting minutes here as well as agendas that we'll want to get, and I think all that would be needed is to check if "Agenda" or "Minutes" is in the string text